src/lua/elrsV3.lua

```diff
@@ -891,6 +891,31 @@ local function setMock()
   deviceIsELRS_TX = true
 end
 
+local function checkCrsfModule()
+  -- Loop through the modules and look for one set to CRSF (5)
+  for modIdx = 0, 1 do
+    local mod = model.getModule(modIdx)
+    if mod and mod.Type == 5 then
+      -- CRSF found
+      checkCrsfModule = nil
+      return 0
+    end
+  end
+
+  -- No CRSF module found, save an error message for run()
+  lcd.clear()
+  local y = 2
+  lcd.drawText(2, y, "  No ExpressLRS", MIDSIZE)
+  y = y + (textSize * 2)
+  lcd.drawText(2, y, " Enable a CRSF Internal")
+  y = y + textSize
+  lcd.drawText(2, y, "   or External module in")
+  y = y + textSize
+  lcd.drawText(2, y, "      Model settings")
```

Comment:
Internal modules are enabled in Radio settings / Hardware :grimacing:

Reply:
Well, yes and no... they are *usually* configured once off in Radio Settings/SYS => Hardware... but other than that... it's Model Settings => ... ie. OFF/CRSF (ON). If it's wrong there, that is a different issue... RTFM is required :rofl:

Reply:
Do we think this is too much?
![image](https://github.com/user-attachments/assets/e5d1ed92-80c5-4cd1-871d-914728e16e3a)

Reply:
I'd say it is correct and fits.

Reply:
Great! I'mma recuse myself due to how much I've taken over and let you do the approval if you don't mind :)